scripts/ci/logcheck/check-restart-logs_test.bats

```diff
@@ -62,32 +52,32 @@ TEST_FIXTURES="${BATS_TEST_DIRNAME}/test_fixtures"
 }
 
 @test "it handles exceptions in > 1 logs" {
```

Comment:
It sounds like the purpose of this test was to handle more than one file argument? Can we use another log here instead of the removed one? 🤔